fatemeh/CSS/CSS Flexbox/style.css

```diff
@@ -0,0 +1,73 @@
+.main {
+  background-color: lightgray;
+}
+
+.row1 {
+  display: flex;
+  align-items: stretch;
+}
+.column1 {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+}
+.column1 > div {
+  background-color: lightblue;
+  padding: 5px;
+  flex-grow: 1;
+}
+.column1 > .A {
+  display: flex;
+  justify-content: left;
+}
```

Comment:
I'm not certain if it's necessary to use child selector here, but it seems like to be the right use of selectors. Well Done! the output is almost completely OK.